app/src/main/java/eu/kanade/tachiyomi/ui/player/PlayerActivity.kt

```diff
@@ -484,11 +484,23 @@ class PlayerActivity : BaseActivity() {
                     }
 
                     override fun onSkipToPrevious() {
-                        changeEpisode(viewModel.getAdjacentEpisodeId(previous = true))
+                        if (playerPreferences.mediaChapterSeek().get()) {
+                            if (player.loadChapters() != emptyList<Chapter>()) {
```

Comment:
player.loadChapters().isEmpty() ?

Reply:
I only did it this way because I copied some other code from chapter seeking but I suppose it's a tad bit cleaner